exe/floe

```diff
@@ -4,29 +4,37 @@
 require "floe"
 require "optimist"
 
+supported_docker_runners = {
+  "docker"     => Floe::Workflow::Runner::Docker, # first is the default
+  "podman"     => Floe::Workflow::Runner::Podman,
+  "kubernetes" => Floe::Workflow::Runner::Kubernetes
+}
+
 opts = Optimist.options do
   version("v#{Floe::VERSION}\n")
+
   opt :workflow, "Path to your workflow json",                 :type => :string, :required => true
   opt :input, "JSON payload to input to the workflow",         :default => '{}'
   opt :credentials, "JSON payload with credentials",           :default => "{}"
-  opt :docker_runner, "Type of runner for docker images",      :default => "docker"
-  opt :docker_runner_options, "Options to pass to the runner", :type => :strings
+  opt :docker_runner, "Type of runner for docker images",      :type => :string, :short => 'r'
+  opt :docker_runner_options, "Options to pass to the runner", :type => :strings, :short => 'o'
+
+  supported_docker_runners.each_key do |runner|
+    opt runner.to_sym, "Use #{runner} to run images (short for --docker_runner=#{runner})", :type => :boolean
+  end
 end
 
-Optimist.die(:docker_runner, "must be one of #{Floe::Workflow::Runner::TYPES.join(", ")}") unless Floe::Workflow::Runner::TYPES.include?(opts[:docker_runner])
+# if they specified a docker_runner, and it is unknown, complain
+Optimist.die(:docker_runner, "must be one of #{supported_docker_runners.join(", ")}") if opts[:docker_runner] && !supported_docker_runners.key?(opts[:docker_runner])
```

Comment:
felt strange to have these defined in another file. but every one was then again listed in this file.

the main change here is only requiring the parameter if it was specified (before, it was defaulted so it was always present)